keyboards/compound/readme.md

```diff
@@ -0,0 +1,15 @@
+# Compound
+
+![Compound](https://imgur.com/0DC9F9d.jpg)
+
+A compact 60% Compound keyboard with fixed Tsangan style layout.
+
+* Keyboard Maintainer: [yumagulovrn](https://github.com/yumagulovrn)  
+* Hardware Supported: Compound PCB
+* Hardware Availability: [GeekHack IC](https://geekhack.org/index.php?topic=111187.0)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you also add instructions on how to reset the controller into the bootloader mode, so new firmware can be flashed?  

Eg, does it have a physical reset button?  Bootmagic?  Reset keycode?

Reply:
Thank you for the comment! Added proper instructions in 4a811a7